.github/workflows/srt.yml

```diff
@@ -108,13 +138,21 @@ jobs:
 
       - name: scripts regression tests
         run: |
+          pwd
+          echo "-----"
+          ls $HOME/work/cime
+          echo "-----"
+          ls $HOME/work/cime/cime
+          echo "-----"
+          ls $HOME/work/cime/cime/components
+          echo "-----"
           mkdir -p $HOME/cesm/scratch
           mkdir -p $HOME/cesm/inputdata
           cd $HOME/work/cime/cime/scripts/tests
           export NETCDF=$HOME/netcdf-fortran
           export PATH=$NETCDF/bin:$PATH
           export LD_LIBRARY_PATH=$NETCDF/lib:$HOME/pnetcdf/lib:$LD_LIBRARY_PATH
-          ./scripts_regression_tests.py --no-fortran-run --compiler gnu --mpilib openmpi --machine ubuntu-latest
+#CAF          ./scripts_regression_tests.py --no-fortran-run --compiler gnu --mpilib openmpi --machine ubuntu-latest
```

Comment:
so we aren't running the test at this time?

Reply:
I commented this out for debugging.  We're suppose to be running the test.

Reply:
Fixed